fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -679,10 +700,31 @@ object Back extends Category(offset = 12, width = 9){
     for(x <- 0 to 256)
     yield makeAttr(s"\u001b[48;5;${x}m", 17 + x)(s"Back.Full($x)")
 
-  val all = Vector(
+  def True(hex: Long) = {
+    val r = hex >> 16
+    val g = (hex & 0x00FF00) >> 8
+    val b = hex & 0x0000FF
+    makeAttr(s"\u001b[48;2;${r};${g};${b}m", 174 + hex)(s"Back.True($r,$g,$b)")
+  }
+
+  def True(r: Int, g: Int, b: Int) = {
+    val i = r << 16 | g << 8 | b
+    makeAttr(s"\u001b[48;2;${r};${g};${b}m", 174 + i)(s"Back.True($r,$g,$b)")
```

Comment:
How I didn't know that, I was thinking interpolations are compile into string concatenation.
`"\u001b[48;2;"+r+";"+g+";"+b+"m"`